### DISCUSSION TITLE
Add: Nickelodeon Canada & Family Channel, Treehouse TV, Universal Kids HD, Bravo, WETA PBS & More

### DISCUSSION THREAD

**Discussion by Westh977** [iptv-org/community#1110] [Q&A]
Dear, IPTV I Want You to Add More US & Canada Kids & Comedy Channels Into Playlist 
List
Nickelodeon Canada
Universal Kids HD East
Nickelodeon West
Family Channel
WETA PBS
Bravo USA
Boomerang USA
Cartoon Network HD USA
Nick Jr. Too UK
Treehouse TV
Cartoon Network Canada
Boomerang Canada
YTV Canada
Can You Please Add Those Channels Into The Playlist With Logos On Them Please If You Reply To Me In Email Let Me Know If You Guys Add It Or Not

**Comment by dtngfan85** [comment]
Nearly all of those channels are under DMCA takedowns from Corus and WBD

WETA doesn't provide an online stream, instead they only have a DRM protected stream of their WETA Metro channel.

**Reply by Pecaquito** [reply]
You can find those channel on this playlist.
https://bit.ly/DOM_m3u

**Reply by dtngfan85** [reply]
It doesn't matter if those channels are available elsewhere, both Corus and WBD have already sent multiple DMCA takedowns to this github repository. Ignoring a DMCA and readding their stations would likely lead to a permanent ban from github. The only thing you're accomplishing is getting those external links onto their lawyers radar.

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
